File: data/imported/knowledge/slides/java/merged/snippets/snippet_075.java
@Test
    void testAddAutoShape() {
        try (var pres = new Presentation()) {
            var slide = blankSlide(pres);
            var shape = slide.getShapes().addAutoShape(ShapeType.RECTANGLE, 50, 50, 200, 100);
            assertThat(slide.getShapes().size()).isEqualTo(1);
            assertThat(shape.getShapeType()).isEqualTo(ShapeType.RECTANGLE);
        }
    }

File: data/imported/knowledge/slides/net/merged/snippets/snippet_075.cs
[Fact]
    public void TestRemoveShape()
    {
        // Removing a shape by reference decreases count.
        using var pres = new Presentation();
        var slide = BlankSlide(pres);
        var s = slide.Shapes!.AddAutoShape(ShapeType.Rectangle, 50, 50, 200, 100);
        slide.Shapes!.AddAutoShape(ShapeType.Ellipse, 300, 50, 150, 150);
        slide.Shapes!.Count.Should().Be(2);
        slide.Shapes!.Remove(s);
        slide.Shapes!.Count.Should().Be(1);
    }